{"timestamp": "2017-11-06 04:04:01.758954", "error_text": "ConnectionError(('Connection aborted.', error(111, 'Connection refused'))) caused by: ProtocolError(('Connection aborted.', error(111, 'Connection refused')))", "error": 1}